Uplift/Models/Equipment.swift

```diff
@@ -35,32 +36,111 @@ struct Equipment: Hashable {
     init(from equipment: EquipmentFields) {
         // Unwrap and convert GraphQL enum value to Swift enum value
         self.accessibility = equipment.accessibility?.value
-        self.equipmentType = equipment.equipmentType.value
         self.facilityId = equipment.facilityId
+        self.muscleGroup = [MuscleGroup](equipment.muscleGroups.compactMap(\.?.value))
```

Comment:
very interesting syntax could you mak the $0 instead. Im a swiftUi noob so i had to look this up
```suggestion
        self.muscleGroup = [MuscleGroup](equipment.muscleGroups.compactMap { $0?.value })
```